ensemble-dss/src/main/java/hec/dss/ensemble/DssDatabase.java

```diff
@@ -35,22 +35,32 @@
  */
 public class DssDatabase implements EnsembleDatabase,MetricDatabase {
     private String dssFileName;
-    Catalog catalog;
+    private Catalog catalog;
     static DateTimeFormatter dssDateFormat = DateTimeFormatter.ofPattern("ddMMMyyyy HHmm");
```

Comment:
Can the three DateTimeFormatters here be make private?

Reply:
yep!